test/ConnectionTest.cpp

```diff
@@ -15,6 +15,16 @@ namespace {
         return connection.advancedQuery().serverVersion;
     }
 
+    int getDispatchedConnectionInfo(const IDBConnection2& dbConnection) {
+        QueryReceiver queryReceiver;
```

Comment:
@Morheit, I suggest to follow Google coding style. In particular naming conventions: https://google.github.io/styleguide/cppguide.html#Naming

Reply:
@malirod Done